engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -219,6 +219,45 @@ paths:
       security:
         - google_oauth:
             - openid
+  '/workflows/{version}/{id}/labels':
+    patch:
+      summary: Add custom labels to an existing workflow.
```

Comment:
Going with: `Add new labels or update values for existing label keys by workflow id.`